packages/eslint-plugin/README.md

```diff
@@ -58,6 +58,19 @@ To use this configuration, add the following to your `.eslintrc.json` file:
 }
 ```
 
+### monorepo-root
+
+The `monorepo-root` configuration is suitable for the root tsconfig.
```

Comment:
seems obvious but should "of a monorepo project" added at the end?

The `monorepo-root` configuration is suitable for the root `tsconfig` file of a monorepo project.